referrer-policy/generic/fetch.js

```diff
@@ -0,0 +1,337 @@
+// Shamelessly copied from: https://github.com/github/fetch/blob/master/fetch.js
+// TODO(krisitijanburnik): See how to include third_party for WPT.
```

Comment:
Done. Passing tests for browsers which don't support fetch. However, there is a subresource-test which will fail on those.